src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -6,31 +6,86 @@
 import io.swagger.annotations.ApiResponses;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MissingServletRequestParameterException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.regex.Pattern;
+
 
 @RestController
 public class GetValidOpenDFCMCase {
     private static final Logger logger = LogManager.getLogger(GetValidOpenDFCMCase.class);
 
+    /**
+     * getValidOpenDFCMCase
+     * <p>
+     * WARNING: Do not modify input parameters. This operation must replicate legacy
+     * system exactly.
+     * <p>
+     * Note: In the event of a unexpected parameter patter this methods will return an HTTP status
+     * code of 200. THIS IS EXPECTED AND WHAT THE LEGACY SYSTEM DOES. PLEASE DO NOT
+     * CHANGE THIS. An error is indicated to the calling system by the -2 int
+     * value in the response structure.
+     *
+     * @param driversLicense
+     * @param surcode
+     */
     @RequestMapping(value = "/getValidOpenDFCMCase",
             produces = {"application/xml"},
             method = RequestMethod.GET)
     @ApiOperation(value = "Generates application callback endpoints for DPS Validation Service", notes = "", tags = {"DPSValidationService"})
```

Comment:
The value for "value" should be something like "Driver Fitness Case Management Validation Service"

Reply:
I think "value" still requires changing. Thanks..

Reply:
changed the value